toolbox/bulk_processing/validators.py

```diff
@@ -238,6 +244,13 @@ def validate(value, **kwargs):
             return
 
         case_id = kwargs['row']['CASE_ID']
+
+        try:
```

Comment:
This could have a unit test for the `mandatory_after_update` validator that checks when it's passed an invalid UUID then the database execute is never called, the same as you added for the other validator

Reply:
Ok, will get on it